manifests/o/oleg-shilo/cs-script/4.8.17.0/oleg-shilo.cs-script.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: oleg-shilo.cs-script
+PackageVersion: 4.8.17.0
+PackageLocale: en-US
+Publisher: oleg-shilo
+PackageName: cs-script
+License: MIT License
+ShortDescription: C# Script execution engine
```

```suggestion
ShortDescription: 'C# Script execution engine'
```
Quotes are needed as `#` is the beginning or a YAML comment